build/compile-validators.js

```diff
@@ -60,6 +60,7 @@ const schemas = {
 };
 
 const ajv = new Ajv({ code: { source: true, esm: true } });
+ajv.addFormat('date-time', /^\d\d\d\d-[0-1]\d-[0-3]\dt(?:[0-2]\d:[0-5]\d:[0-5]\d|23:59:60)(?:\.\d+)?(?:z|[+-]\d\d(?::?\d\d)?)$/i);
```

Comment:
Can we define the regex in `definitions.json` so we only need to add these regex declaratively without needing awareness to js file.

Reply:
I'm gonna try to find a way to do it, it's a good idea.

I had no idea that round-tripping a RegExp to string and back in JavaScript is this involved: https://stackoverflow.com/questions/39154255/converting-regexp-to-string-then-back-to-regexp

Reply:
@duncanmak, I have gone ahead and moved it into `definitions.json`. Let me know if you have any questions.